test/server/server_test.cc

```diff
@@ -601,6 +601,7 @@ TEST_P(ServerInstanceImplTest, BootstrapRuntime) {
   EXPECT_EQ("bar", server_->runtime().snapshot().get("foo"));
   // This should access via the override/some_service overlay.
   EXPECT_EQ("fozz", server_->runtime().snapshot().get("fizz"));
+  EXPECT_EQ("foobar", server_->runtime().snapshot().getLayers()[3]->name());
```

Comment:
@LisaLudique @htuch the original bug was in the validation server? Does this test give us validation server coverage? Also, @asraa, I thought we fuzz validation config. Is that not the case yet? I'm surprised we haven't seen this in the fuzz results.

Reply:
Probably not. We do have some issue like this in one of the fuzz bugs IIRC.

Reply:
Yeah, I believe this is it: https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=16870
I hadn't gone through debugging this yet, but I can definitely add this testcase in for coverage.